examples/v1beta1/taskruns/tekton-bundles.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  generateName: remote-task-reference-
+spec:
+  taskRef:
+    name: hello-world
+    bundle: registry.hub.docker.com/ptasci67/test-oci@sha256:7a8e6b41d37e16c86dc38fbb395673fe496edb9ac759aee135df488b55b4c732
```

Comment:
We should move this to a Tekton-owned repo, maybe gcr.io/tekton-nightly?

Reply:
Yeah what we'll probably end up doing is moving this to a real Go test that pushes a bundle image first then references it, to ensure it's there and up-to-date.

We can stick with this for now, but we should have a TODO to clean it up soon, in case Docker deletes the image or something 😅

Reply:
Will do.